packages/clay-card/src/CardWithFolder.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayCard from './Card';
+import ClayForm from '@clayui/form';
+import ClayIcon from '@clayui/icon';
+import ClaySticker from '@clayui/sticker';
+import React from 'react';
+
+interface IProps {
+	actions?: [];
+	onSelectChange?: (val: boolean) => void;
+	spritemap: string;
+	selected?: boolean;
+	title: string;
+}
+
+export const ClayCardWithFolder: React.FunctionComponent<IProps> = ({
+	onSelectChange,
+	selected = false,
+	spritemap,
+	title,
+}) => {
+	const content = (
+		<ClayCard.Body>
+			<div className="autofit-col">
+				<ClaySticker inline>
+					<ClayIcon spritemap={spritemap} symbol="folder" />
```

Comment:
These will be configurable? I would assume the icon for `CardWithFolder` would always be a folder icon? If we make it changeable, then maybe we just need to make a generic high-level for all use cases?

Reply:
Hmm, we left it configurable in v2 to avoid having to add at some point in the future... But since it is a high-level for a very specific case, maybe it is not necessary, we can confirm with the Lexicon. I remember that for the user's card we allowed the icon to be configurable but the Lexicon later decided to be just the icon of `User` and Victor Valle asked to leave it configurable on Clay.

Reply:
hmm maybes its best to keep it specific for now, and then if needed, we can make it more generic in a future version.

Reply:
Just a correction of what I said, before the card user could be letters, icons and images, but still allow you to use images and icons (just the User for now...). So we still have to leave it configurable for the `CardWithUser` case.

I was checking the documents again and we have to make the `CardWithFolder` icon configurable.

https://docs.google.com/document/d/1st2moaRVphrsFrDJUz-ic26aH-V46YkQam75X6BHPY0/edit#heading=h.l82je57kortn

Reply:
I'm gonna switch it up a bit and aim specifically for CardWithUser in this PR. I think getting that fully worked through will make the others easier to add.